Sources/FoundationEssentials/ProcessInfo/ProcessInfo.swift

```diff
@@ -117,7 +117,11 @@ final class _ProcessInfo: Sendable {
     }
 
     var processIdentifier: Int32 {
```

Comment:
Is there a reason to make this `Int32` when it is almost universally `UInt32`?

Reply:
The public `ProcessInfo.processIdentifier` API today is of the type `Int32` so we need to maintain that type here for source compatibility (and ABI compatibility on Darwin)